src/components/weather.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import axios from "axios";
+
+const OPEN_WEATHER_API_KEY = "3540653b10b634e1bdacc26d27f1e270";
```

Comment:
Please delete this key on the open weather profile ASAP. If you don't this might cost you money